neqo-transport/src/recovery.rs

```diff
@@ -210,7 +210,6 @@ impl LossRecoverySpace {
     fn remove_ignored(&mut self) -> impl Iterator<Item = SentPacket> {
         // The largest acknowledged or loss_time should still be unset.
         // The client should not have received any ACK frames when it drops 0-RTT.
-        assert!(self.largest_acked.is_none());
```

Comment:
I think that we can assert that either this is true or the PNSpace != PNSpace::ApplicationData